2019/2019-10/2019-10-18-pulsar-weekly.md

```diff
@@ -1,6 +1,8 @@
-[Community Weekly Update] 2019-10-07 ~ 2019-10-18
-
-This is the first weekly community update, which helps you quickly capture Pulsar's highlights and spot trends over last week, meanwhile strengthen the communication and connection within the Pulsar family.
+---
+title: "Community Weekly Update"
```

Comment:
I would suggest keeping the original content as well. Because it can be used for posting the content to the mailing list.  You can add tags for the website.